src/DocumentFormat.OpenXml.Framework/OpenXmlPartWriterSettings.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace DocumentFormat.OpenXml;
+
+/// <summary>
+/// Settings for the OpenXmlPartWriter.
+/// </summary>
+public class OpenXmlPartWriterSettings
+{
+#if FEATURE_ASYNC_SAX_XML
+    /// <summary>
+    /// Gets or sets a value indicating whether asynchronous OpenXmlPartWriter methods can be used.
+    /// </summary>
+    public bool Async { get; set; }
+#endif
+
+    /// <summary>
+    /// Gets or sets a value indicating whether the OpenXmlPartWriter should check to ensure that all characters in the document conform to the "2.2 Characters" section of the W3C XML 1.0 Recommendation.
```

Comment:
```suggestion
    /// Gets or sets a value indicating whether the <see cref="OpenXmlPartWriter" /> should check to ensure that all characters in the document conform to the "2.2 Characters" section of the W3C XML 1.0 Recommendation.
```

Do you have a link to that part of the docs?

Reply:
Is that really what the "CloseOutput" property does?